doc/zmq_socket_monitor_versioned.txt

```diff
@@ -188,7 +188,7 @@ The event value is the status code returned by the ZAP handler (i.e. 300,
 Supported events (v2)
 ----------------
 
-ZMQ_EVENT_PIPE_STATS
+ZMQ_EVENT_PIPES_STATS
 ~~~~~~~~~~~~~~~~~~~~
```

Comment:
the length of the line below needs to match or rendering goes off

Reply:
fixed